carma_nav2_port_drayage_demo/README.md

```diff
@@ -1,5 +1,70 @@
 # `carma_nav2_port_drayage_demo` package
 
-This package implements port drayage integration with a Nav2-based autonomy stack. The package's node listens for
-incoming port drayage variants of `MobilityOperation.msg` messages then sends the target location to the Nav2 Waypoint
+This package integrates Port Drayage operations with a Nav2-based autonomy stack. The package's node listens for
+incoming port drayage messages of type `carma_v2x_msgs/msg/MobilityOperation` and then sends the target location to the Nav2 Waypoint
 Follower via a ROS 2 action call.
+
+## Sending Mobility Operation Messages
+The definition of the `carma_v2x_msgs/msg/MobilityOperation` is as follows:
+
+```
+# MobilityOperation.msg
+
+# This message is used by plugins to send necessary parameters to other
+# CAVs at any desired frequency in order to accomplish a collaborative plan
+
+# standard header for all mobility messages
+carma_v2x_msgs/MobilityHeader  m_header
+	#
+	string  sender_id
+	string  recipient_id
+	string sender_bsm_id
+	string  plan_id
+	uint64  timestamp
+
+# strategy string to indicate which strategy type this message is used for
+string                   strategy
+
+# strategy parameters specified by each individual plugin/component
+# the maximum length of this string is 100
+string                   strategy_params
+```
+
+For the purpose of this demo, the only necessary fields to populate when sending instructions to the vehicle are `strategy` and `strategy_params`. The `strategy` field should always be set to `carma/port_drayage`. The `strategy_params` field should be set to a JSON string that follows the format detailed [here](https://usdot-carma.atlassian.net/wiki/spaces/CRMFRT/pages/2006482998/Detailed+Design+-+Port+Drayage+Plugin+V2X-Hub#Communication). For example, to send a Mobility Operation Message instructing the vehicle to pick up cargo with the ID "`C1T_CARGO`" at `(x,y)` coordinates `(3.8, 0.5)` using the ROS2 CLI, run the command:
```

Comment:
People outside Leidos don't have access to see this page. It might be good to add that description here too, or a link to an external source if we have one.

Reply:
Good catch! I just replaced the link with the relevant documentation on the JSON format.